book/forms.rst

```diff
@@ -1048,7 +1051,8 @@ that will house the logic for building the task form::
             $builder
                 ->add('task')
                 ->add('dueDate', null, array('widget' => 'single_text'))
-                ->add('save', 'submit');
+                ->add('save', 'submit')
```

Comment:
IIRC, that best practice was about to be removed (or removed already). However, I'm still +1 on this change.